dashboard/db/migrate/20170120193705_change_id_column_type.rb

```diff
@@ -0,0 +1,9 @@
+class ChangeIdColumnType < ActiveRecord::Migration[5.0]
+  def up
+    change_column :activities, :id, :bigint
+  end
+
+  def down
+    change_column :activities, :id, :integer
```

Comment:
I agree with Brad. I'd rather have an empty `down` clause. I don't like having code sitting around that we have reason to believe may cause substantial data loss. Happy to being overruled if someone can say why what's currently here is better.